zerver/tests/test_upload.py

```diff
@@ -1305,7 +1306,9 @@ def test_copy_avatar_image(self) -> None:
             self.client_post("/json/users/me/avatar", {"file": image_file})
 
         source_user_profile = self.example_user("hamlet")
-        target_user_profile = self.example_user("iago")
+        target_user_profile = do_create_user(
+            "user@zulip.com", "password", get_realm("zulip"), "user", acting_user=None
+        )
```

Comment:
We create the onboarding_step in `populate_db` for `iago`. So, `copy_default_settings()` results in integrity error when we `copy_onboarding_steps`. Hence this change.